addons/mail/models/mail_render_mixin.py

```diff
@@ -270,7 +270,7 @@ def _render_template_qweb(self, template_src, model, res_ids,
             variables['object'] = record
             try:
                 render_result = self.env['ir.qweb']._render(html.fragment_fromstring(
-                    template_src, create_parent='div'), variables, raise_on_code=is_restricted)
+                    template_src, create_parent='div'), variables, raise_on_code=is_restricted, **(options or {}))
```

Comment:
As options is a generic kwargs given to Odoo rendering methods, I would whitelist what is given to fragment_fromstring. I am unsure it supports whatever stuff we could give as options in _render_template_qweb.

Reply:
Correct me if I'm wrong as I'm reading this on my phone on my off day (can't help myself) but the options are passed to `_render`, not `fragment_fromstring`, and I would expect `QWeb`'s `_render` to support the options of `_render_template_qweb`.

Reply:
Arf, indeed. Then I propose to better multi-line it, current writing is misleading :D